src/main/java/frc/robot/Intake.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.VictorSP;
+import edu.wpi.first.wpilibj.Solenoid;
+
+public class Intake {
+
+    //ESC declarations
+    private final VictorSP intakeMotor; 
```

Comment:
While we are correcting whitespace, let us also ensure there is a space between the `//` and the body of the comment.
Example: `// ESC declarations`